src/client/src/client/cluster.rs

```diff
@@ -71,23 +72,14 @@ impl Fluvio {
         let (socket, config, versions) = inner_client.split();
         let socket = AllMultiplexerSocket::shared(socket);
 
-        let spu_pool = OnceCell::new();
-        Ok(Self {
+        let spu_pool = SpuPool::start(config.clone(), &socket).await?;
```

Comment:
restore lazy start since we don't want need to start spu pool only when consumer/producer requested it